src/browser/modules/EditorInfo/DisplayNodeDetails.jsx

```diff
@@ -48,7 +49,13 @@ const LabelSection = props => {
   return (
     <div>
       <DrawerSection>
-        <DrawerSubHeader>Labels</DrawerSubHeader>
+        <DrawerSubHeader>
+          Labels
+          <AddLabel
```

Comment:
addLabel component inside sub header?

Reply:
Yes, because it is the part of the header.